include/revng/Support/GzipTarFile.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/Support/Error.h"
+#include "llvm/Support/MemoryBuffer.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include "revng/Support/Assert.h"
+
+struct archive;
```

Comment:
> libarchive is a C library?

Yes

> wrap C stuff in extern "C"

I didn't because [libarchive already does it](https://github.com/libarchive/libarchive/blob/master/libarchive/archive.h#L144)

Reply:
I know, I'm talking about this foward declaration.

Reply:
Ah sorry, misread

Reply:
Done